doc/mgr/dashboard.rst

```diff
@@ -156,15 +156,15 @@ which corresponds to all available IPv4 and IPv6 addresses.
 These defaults can be changed via the configuration key facility on a
 cluster-wide level (so they apply to all manager instances) as follows::
 
-  $ ceph config set mgr mgr/dashboard/server_addr $IP
-  $ ceph config set mgr mgr/dashboard/server_port $PORT
+  $ ceph config set mgr/dashboard/server_addr $IP
```

Comment:
please see http://docs.ceph.com/docs/master/rados/configuration/ceph-conf/#commands 

`mgr` after `config set` is not superflous. it like the `<section>` in `ceph.conf` which limits the scope of settings. and it makes it more explicit that `mgr/*` settings only affects mgr.

Reply:
Arrghh, had 3 version of the docs open, and still looked at the wrong ones.
You are correct.